management-server/Makefile

```diff
@@ -0,0 +1,22 @@
+#   Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
we won't need this once we moved proto files to adapter. Include these into protogen.sh script